mk/spksrc.common.mk

```diff
@@ -15,7 +15,7 @@ RUN = cd $(WORK_DIR)/$(PKG_DIR) && env $(ENV)
 
 # Pip command
 PIP ?= pip
-PIP_WHEEL = $(PIP) wheel --no-binary :all: --no-deps --requirement $(WORK_DIR)/wheelhouse/requirements.txt --wheel-dir $(WORK_DIR)/wheelhouse --build-dir $(WORK_DIR)/wheelbuild
```

Comment:
Please leave `--no-deps` and use `pip freeze` command to generate requirement file with module explicit versions. That is required to have reproduceable package generation even many months and updates later. 
Refer to https://github.com/SynoCommunity/spksrc/wiki/Using-wheels-to-distribute-Python-packages for details

Reply:
Alright, that makes sense.